html/portfolio.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="../stylesheets/normalize.css">
+    <link rel="stylesheet" href="../stylesheets/styles.css">
+    <link rel="stylesheet" href="../stylesheets/styles_portfolio.css">
+    <link rel="stylesheet" href="../stylesheets/font-awesome-4.7.0/css/font-awesome.css">
+  </head>
+
+  <body>
+    <header class="header-and-footer">
+      <a id="home" href="index.html"> Home </a>
+      <nav>
+        <ul>
+          <li><a href="about.html"> About </a></li>
+          <li><a href="portfolio.html"> Portfolio </a></li>
```

Comment:
when I click on this link twice in a row I get a ERR_FILE_NOT_FOUND .  not sure if it's just on my computer, but happens consistently.